kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/DeploymentUtils.java

```diff
@@ -235,6 +241,31 @@ public static void registryCredentialsSecret(String namespace) {
         }
     }
 
+    /**
+     * Collect cluster info.
+     *
+     * @param namespace the namespace
+     * @param testClassName the test class name
+     * @param testMethodName the test method name
+     */
+    public static void collectClusterInfo(String namespace, String testClassName, String testMethodName) {
+        String formattedDate = DateTimeFormatter.ofPattern("yyyy_MM_dd_HH_mm_ss")
+                .withZone(ZoneOffset.UTC)
+                .format(Instant.now());
+        List<String> executableCommand = List.of(cmdKubeClient(namespace).toString(), "cluster-info", "dump",
+                "--namespaces", String.join(",", namespace, Constants.KAFKA_DEFAULT_NAMESPACE),
```

Comment:
Do we need/want a dump of the strimzi operator namespace here as well?

Reply:
If Strimzi operator is installed by us, we install it in the KAFKA_DEFAULT_NAMESPACE, if it is installed by someone else, we cannot know where it is installed.

Another option I see is to remove the `--namespaces` parameter and always collect all the info from all namespaces, but it could be huge in Openshift for example.

Reply:
How it is makes sense to me. Let's not complicate it any further right now. We can do something else if we need to later